SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -220,7 +273,7 @@ function Set-Secret
         }
     }
 
-    if ($Secret -is [hashtable]){
+    if ($Secret -is [System.Collections.Specialized.OrderedDictionary] -or $Secret -is [hashtable]) {
```

Comment:
Why did you add ordereddictionary here?

Reply:
@TylerLeonhardt 
I forgot to add a comment.
In fact, I forgot about it and had to check in the code.

It is a missing check from : 
https://github.com/TylerLeonhardt/SecretManagement.LastPass/pull/22

Since that PR implemented the ordered dictionary to preserve the order of parameters, everywhere that checked previously for a hashtable now need to check for an ordered dictionary.
(All other instances of this were covered by aforementionned PR)

That being said, I also kept legacy in case someone wanted to build from the ground up a secret using a hashtable and push it to LastPass.

Reply:
So [Ordereddictionary](https://docs.microsoft.com/en-us/dotnet/api/system.collections.specialized.ordereddictionary?view=net-5.0) and [hashtable](https://docs.microsoft.com/en-us/dotnet/api/system.collections.hashtable?view=net-5.0) inherit from [IDictionary](https://docs.microsoft.com/en-us/dotnet/api/system.collections.idictionary?view=net-5.0) so if you replace this with just one `-is [System.Collections.IDictionary]` it should still work.

Reply:
This also means that if a user passes in anything that inherits from IDictionary, we will support it :)

Reply:
Ohhhhh 👍 Done.